### PR TITLE
allows marines to build on grass and wet grass

### LINKED ISSUE #3838: You cannot build barricades on grass
## Testmerges:
None that matter
## Reproduction:
Due to a change in https://github.com/tgstation/TerraGov-Marine-Corps/pull/1203 the barricades are not able to be constructed on grass or other non normal open/floor turf.
Examination of the code before and after would reveal that they were buildable on grass or even in the river as it is still on the CM13 codebase.
https://github.com/tgstation/TerraGov-Marine-Corps/blame/dc5cedb1fe8903e7abcca24c43423b32702fb157/code/game/objects/items/stacks/stack.dm#L226

### PR DESCRIPTION
**Pull request by MacBlaze1**
## About The Pull Request

this PR allows marines to build structures on grass and wet grass
Closes #3838
## Why It's Good For The Game

I was told to fix this by highly experienced players and I believe this is a fix which will reduce headaches for whiskey outpost. 

## Changelog
:cl:
fix: fixed marines being unable to build on grass and wet grass (most useful on Whiskey Outpost, but also elsewhere)
/:cl:

### CODE REVIEW

File: code/__DEFINES/is_helpers.dm [hunk 1]
```diff
@@ -11,6 +11,8 @@
 
 #define isopenturf(A) (istype(A, /turf/open))
 
+#define isopengroundturf(A) (istype(A, /turf/open/ground/jungle) || istype(A,/turf/open/ground/grass))
```
Comment (Hughgent) [inline code/__DEFINES/is_helpers.dm:14]:
```suggestion
#define isopengroundturf(A) (istype(A, /turf/open/ground/jungle) || istype(A, /turf/open/ground/grass))
```

**Comment by spookydonut** [comment]
balance issue is why this is a thing, the places with grass aren't meant to be build on.  if you want marines to build places mapedit to change the it to not-grass

**Comment by psykzz** [comment]
I agree that balance is a concern but not allowing grass restricts mapping creativity.
Im down to allow this for now and see how it affects balance, i think only LV would be affected

**Comment by neohenry** [comment]
After marines got this PR, I sought out to have xeno the same ability, that is to construct resin stuff in grass. It made sense to have xenos achieve the same level. That being said, I asked other members about the balance on xenos.

Here are the issues if xenos can build on grass (source: Discord server in #balance):

image
image

However, JPR countered these arguments in that marines could easily deconstruct vines and other tiles:

image
image
image

**Comment by neohenry** [comment]
The PR was initially made for WO so that marines could construct more lines of cades, but it had the unintended consequences of extending this to LV-624's designs for both marines and xenomorphs.

**Pull request merged**